int32 fr_pwm
int32 lr_pwm
int32 turret_pulse_width
string xAxisDirection
string yAxisDirection
int32 movingRobot
